atomics/performance_tests/kokkos_based/PerfTestAtomics.hpp

```diff
@@ -423,8 +418,7 @@ double test_atomic_perf_random_neighborhood(int N,
   int errors = 0;
   Kokkos::parallel_reduce(
       "desul::Tests::PerfRandNeigh::Check",
-      Kokkos::RangePolicy<Kokkos::DefaultHostExecutionSpace>(
-          exec_space, 0, result_device.extent(0)),
+      Kokkos::RangePolicy(exec_space, 0, result_device.extent(0)),
```

Comment:
I think this one was correct. `result_device` and `org_dst_values` are host Views.

Reply:
it compiles, but I still thought we should have template argument deduction solve it for us

Reply:
Yeah using CTAD is better.